src/react_native/gesture.cljs

```diff
@@ -95,13 +95,25 @@
             (into [{:title title :header? true}] data))
    sections))
 
+(defn find-sticky-indices
+  [data]
+  (->> data
+       (map-indexed (fn [index item] (if (:header? item) index nil)))
+       (filter (complement nil?))
+       (vec)))
+
 (defn section-list
-  [{:keys [sections render-section-header-fn render-fn] :as props}]
+  [{:keys [sections sticky-section-headers-enabled render-section-header-fn render-fn]
+    :or   {sticky-section-headers-enabled true}
+    :as   props}]
```

Comment:
For section lists, sticky headers are enabled by default. This implementation uses `FlatList` under the hood (from gesture handler) because the normal section list from react native wouldn't work (the gestures do not work) in the context of a bottom sheet (this case).

Reply:
Ah I see, that makes sense, thanks for the explanation 🙏